audplot/core/api.py

```diff
@@ -645,3 +645,137 @@ def spectrum(
     )
 
     return image
+
+
+def waveform(
+        x: np.ndarray,
+        *,
+        text: str = None,
+        color: typing.Union[str, typing.Sequence[float]] = '#E13B41',
+        background: typing.Union[str, typing.Sequence[float]] = (0, 0, 0, 0),
```

Comment:
I think I would change to `'#FFFFFF'` to use the same format as we do with `color`.

Reply:
`'#FFFFFF'` would be identical with `(0, 0, 0, 1)`. We are setting the background to transparent as default.
Would you prefer white instead?

Reply:
Ah I see. Maybe change to `'#FFFFFF00'` then?

Reply:
Updated accordingly.